vendor/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -397,6 +403,12 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 			continue
 		}
 	}
+
+	// output empty list warning when no resources to display
+	if len(objs) == 0 {
+		fmt.Fprintf(os.Stderr, "%s\n", "There are no resources to display.")
```

Comment:
BTW we need to make sure `Stdout`, `Stderr`, etc are passed to commands by the root command and that we use what was passed in every command. The entire CLI structure is pretty screwed in that sense. I'm addressing it in one of my PRs.